pimd/pim_instance.c

```diff
@@ -50,6 +51,8 @@ static void pim_instance_terminate(struct pim_instance *pim)
 
 	pim_rp_free(pim);
 
+	pim_bsm_data_free(pim);
```

Comment:
let's call this function name mirrored to the init?  pim_bsm_proc_free();

Reply:
shall change it to pim_bsm_proc_free